README.md

```diff
@@ -44,6 +44,7 @@ kexec-bundle | same as before, but it's just an executable
 lxc | create a tarball which is importable as an lxc container, use together with lxc-metadata
 lxc-metadata | the necessary metadata for the lxc image to start, usage: lxc image import $(nixos-generate -f lxc-metadata) $(nixos-generate -f lxc)
 openstack | qcow2 image for openstack
+proxmox | [VMA}(https://pve.proxmox.com/wiki/VMA) file for proxmox
```

Comment:
@Lassulus Should be `proxmox | [VMA](https://pve.proxmox.com/wiki/VMA) file for proxmox` (closing `]` after VMA)

![image](https://user-images.githubusercontent.com/9866621/139864910-b2e9f947-0272-4881-a751-c6f8feb446f4.png)

Reply:
whoops